tests/unit/test.watcher.js

```diff
@@ -207,3 +208,57 @@ describe('watcher', () => {
   });
 
 });
+
+describe('--watch-ignored is passed in', () => {
```

Comment:
Nit, would you mind to move this describe inside the `describe('watcher', () => { ... });` block, at the end on it (my apologies I didn't notice this group was outside out that block until now).